doc/cephfs/cephfs-mirroring.rst

```diff
@@ -398,6 +398,39 @@ created, the mirror daemon will clear the failed state upon successful synchroni
 When mirroring is disabled, the respective `fs mirror status` command for the file system
 will not show up in command help.
 
+Exported Metrics
+----------------
+
+1. cephfs_mirror_fs_mirror_peers     : Mirroring Peers
+                                       The number of peers involved in mirroring.
```

Comment:
Suggest removing the terminal periods here and below where present

Reply:
done